src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,114 @@
+import { IconCode, IconHeart, IconStar } from '@tabler/icons-react';
+import { useTranslation } from 'react-i18next';
+import { Box, Container, SimpleGrid, Text } from '@mantine/core';
+import { theme } from '@/theme';
+import { Link } from '../Link/Link';
+import { Button } from '../UI/Button/Button';
+import { Divider } from '../UI/Divider/Divider';
+import { Typography } from '../UI/Typography/Typography';
+import { footerStyles } from './styles';
+import styles from './styles.module.css';
+
+export default function Footer() {
+  const { t } = useTranslation();
+
+  const QUICK_LINKS = [
+    {
+      href: '#',
+      label: t('footer.About'),
+    },
+    {
+      href: '#',
+      label: t('footer.Scanner'),
+    },
+    {
+      href: '#',
+      label: t('footer.Statistics'),
+    },
+    {
+      href: '#',
+      label: t('footer.Documentation'),
+    },
+  ];
+  const COMMUNITY_LINKS = [
+    {
+      href: '#',
+      label: t('footer.Contribute'),
+    },
+    {
+      href: '#',
+      label: t('footer.Issues'),
+    },
+    {
+      href: '#',
+      label: t('footer.FeatureRequests'),
+    },
+    {
+      href: '#',
+      label: t('footer.Discord'),
+    },
+  ];
+
+  return (
+    <>
+      <Divider />
+      <Container style={{ ...footerStyles.container }}>
+        <SimpleGrid
+          spacing='xl'
+          cols={{ base: 1, sm: 1, md: 3 }}
+          style={{ ...footerStyles.linkBoxWrapper }}
+        >
+          <Box>
+            <Text

```

Comment:
WE have a typography component, is it possible to achieve the same thing with the component?

Reply:
Its ok to merge it like this, I just wonder but cant you style this via `styles.ts` file? without the component prop